### ISSUE TITLE
How to query returns and cancelled orders?

### ISSUE THREAD

**Issue by Thangthebrandlab** [allegro/allegro-api#10466]
Hi,

Is there an effective way to query all orders **except** returns and cancelled orders from the checkout forms endpoint? The default route seems to load all orders regardless it's cancelled or returned. Thanks!

**Comment by MartaNowaczyk** [comment]
No, but from what I understand by returns you mean customer order returns?

**Comment by Thangthebrandlab** [comment]
Yes, customer order returns and cancelled orders.

**Comment by Thangthebrandlab** [comment]
Hi,

Thank you for checking this fact in with me. How would you get which orders that's cancelled or returns form the API? Thanks.

**Comment by MartaNowaczyk** [comment]
From the checkout forms endpoint you can download only canceled orders - [GET /order/checkout-forms?status=CANCELLED](https://developer.allegro.pl/documentation/#operation/getListOfOrdersUsingGET). You cannot retrieve a list of orders that have been returned by the customer, but you can retrieve a list of returns by [GET /order/customer-returns](https://developer.allegro.pl/documentation/#operation/getCustomerReturns).
